net-misc/cadaver/cadaver-0.23.3.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A command line WebDAV client"
+DESCRIPTION="It supports file upload, download, on-screen display, namespace \
+operations (move/copy), collection creation and deletion, and locking \
+operations."
+HOMEPAGE="http://www.webdav.org/cadaver/"
+COPYRIGHT="1998-2008 Joe Orton
+	2002-2003 GRASE Lab, UCSC"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://www.webdav.org/cadaver/cadaver-$portVersion.tar.gz"
+CHECKSUM_SHA256="fd4ce68a3230ba459a92bcb747fc6afa91e46d803c1d5ffe964b661793c13fca"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	cadaver = $portVersion
+	cmd:cadaver
+	"
+REQUIRES="
+	haiku
+	lib:libexpat
+	lib:libiconv
+	lib:libintl
+	lib:libncurses
+	lib:libreadline
+	lib:libssl
+	lib:libxml2
+	lib:libzlib
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libexpat
+	devel:libiconv
+	devel:libintl
+	devel:libncurses
+	devel:libreadline
+	devel:libssl
+	devel:libxml2
+	devel:libzlib
+	"
+BUILD_PREREQUIRES="
+	cmd:automake
+	cmd:bison
+	cmd:gcc
+	cmd:libtoolize
+	cmd:make
+	cmd:pkg_config
+	"
+
+BUILD()
+{
+	libtoolize -fci
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
```

Comment:
Since the `-g` flag is being passed to the compiler, the runtime has debug symbols.
That's why it is quite big (856 KiB on x86_gcc2). So there are a few options:
* Use install-strip:
```diff
-	make install
+	make install-strip
```

* **_or_** ask haikuporter to create a debuginfo package:
```diff
 	cmd:pkg_config
 	"
+
+defineDebugInfoPackage cadaver \
+	"$binDir"/cadaver
 
 BUILD()
```
I think @pulkomandy prefers the install-strip method. That's OK because this recipe does not provide any lib.

Reply:
Done